tailwind.config.js

```diff
@@ -39,7 +39,27 @@ module.exports = {
         8: '8%',
         42: '42%',
       },
+      fontSize: {
+        'heading-1': ['1.5rem', { fontWeight: 700 }],
+        'heading-2': ['1.375rem', { fontWeight: 700 }],
+        'heading-3': ['1.125rem', { fontWeight: 700 }],
+        'bold-body': ['1rem', { fontWeight: 700 }],
```

Comment:
super small detail, but this text class is labeled in the [figma](https://www.figma.com/file/HOeEUn2qt2nP2EvklLLUHZ/Wireframes?node-id=769%3A144&mode=dev) as "Body bold", so this should be named 'body-bold'